first review

gustavo-maurina opened the pull request.

chrishughallen commented on src/App.js:
```diff
@@ -1,20 +1,26 @@
+import { nanoid } from 'nanoid';
```
I assume this change is because it's best practice to have all library imports at the top level?

gustavo-maurina replied:
Correct!

chrishughallen commented on src/App.js:
```diff
@@ -1,20 +1,26 @@
-import { nanoid } from 'nanoid';
-import Confetti from 'react-confetti'
+
+const LETTERS = ['A', 'B', 'C', 'D', 'E', 'F']
```
makes sense!

gustavo-maurina replied:
In this case, it's not so relevant in terms of code maintenance since these letters won't change in the future(I think so). But still, it makes for better readability anyways

chrishughallen commented on src/App.js:
```diff
@@ -31,19 +37,9 @@ function App() {
-    if (swatch.color === winningColor) {
-      setWin(true)
-    }
+    if (win) return false
```
I see you don't like brackets :P 

Is there any performance benefit to removing them? Or you just like the way it looks?

gustavo-maurina replied:
Not a big fan haha, I think that linear code looks better and is better to read than block code, as long as it doesn't breaks the 80-120 col limit

chrishughallen commented on src/App.js:
```diff
@@ -59,25 +55,22 @@ function App() {
 
-  let colors = swatches.map((swatch) => {
-    return(
+  let colors = swatches.map((swatch) => (
```
I guess the return here is implied?

gustavo-maurina replied:
Exactly, on a map function, if you're not doing any logic inside your callback, the return is not needed. But this is not a written rule, I just like it better this way, at the end, is up to you

chrishughallen commented on src/ColorBlock.js:
```diff
@@ -1,13 +1,15 @@
-    backgroundColor: props.color
+
+
+export default function ColorBlock({color, boxColor, checkGuess, showHex}) {
```
I forgot about destructing props, thanks for the reminder!

gustavo-maurina replied:
It's good to have someone look at our code, I always see the opportunity to destruct in other people's code, but often forget about it on my code too

gustavo-maurina commented:
Other than your comments.

-  I moved `useEffect` to be above the functions on your component, because it feels more natural to find things that happen on render at the top of your component. 
- Removed the `if-else if` block, since you were returning inside of them, they were not doing much but making the code more _blocky_.
- changed `let` for `const` where it was suitable
